vnext/Shared/DevSettings.h

```diff
@@ -31,8 +31,9 @@ enum class JSIEngineOverride : int32_t {
   Hermes = 3, // Use the JSIExecutorFactory with Hermes
   V8 = 4, // Use the JSIExecutorFactory with V8
   V8Lite = 5, // Use the JSIExecutorFactory with V8Lite
+  V8Napi = 6, // Use the JSIExecutorFactory with V8 via ABI-safe NAPI
```

Comment:
Just checking if it should be using the new name: `NodeApi`. See https://nodejs.medium.com/renaming-n-api-to-node-api-27aa8ca30ed8

Reply:
Sigh, that does read much better.
Unfortunately, we need to keep this enum in sync with internal sources' replica, so will leave this refactoring for later.

Reply:
On second thought, we can easily rename the duplicate internal source with that enum, currently unused.

I'll rename it on this PR too :)

Reply:
@vmoroz What's best for this value?
- `V8Node`?
- `V8NodeApi`?